dev/com.ibm.websphere.appserver.features/visibility/public/appSecurity-4.0/io.openliberty.appSecurity-4.0.feature

```diff
@@ -12,19 +12,21 @@ IBM-API-Package: jakarta.security.enterprise; type="spec", \
 
 IBM-ShortName: appSecurity-4.0
 Subsystem-Name: Application Security 4.0
--features=io.openliberty.cdi-3.0, \
- io.openliberty.expressionLanguage-4.0, \
- com.ibm.websphere.appserver.security-1.0, \
- com.ibm.websphere.appserver.servlet-5.0, \
- com.ibm.websphere.appserver.eeCompatible-9.0
--bundles=io.openliberty.jakarta.security.2.0; location:=dev/api/spec/; mavenCoordinates="jakarta.security.enterprise:jakarta.security.enterprise-api:2.0.0-RC2", \
- io.openliberty.security.jakartasec.2.0.internal, \
- io.openliberty.security.jakartasec.2.0.internal.cdi, \
- io.openliberty.security.authentication.internal.filter, \
- io.openliberty.security.authentication.internal.tai, \
- io.openliberty.security.sso.internal, \
- io.openliberty.jakarta.authentication.2.0; location:=dev/api/spec/; mavenCoordinates="jakarta.security.auth.message:jakarta.security.auth.message-api:2.0.0-RC1", \
- io.openliberty.security.jaspic.2.0.internal
+-features=\
+  io.openliberty.cdi-3.0, \
+  io.openliberty.expressionLanguage-4.0, \
+  com.ibm.websphere.appserver.security-1.0, \
+  com.ibm.websphere.appserver.servlet-5.0, \
+  com.ibm.websphere.appserver.eeCompatible-9.0
+-bundles=\
+  io.openliberty.jakarta.security.2.0; location:=dev/api/spec/; mavenCoordinates="jakarta.security.enterprise:jakarta.security.enterprise-api:2.0.0", \
```

Comment:
I think you should consider creating a private feature for the API rather than just pulling the bundle in directly.

Reply:
I made this change for both the `jakarta.security.enterprise:jakarta.security.enterprise-api:2.0.0` and `jakarta.security.auth.message:jakarta.security.auth.message-api:2.0.0-RC1` APIs.